src/cclib/method/orbitals.py

```diff
@@ -39,16 +41,17 @@ def __repr__(self):
 
     def closed_shell(self):
         """Return Boolean indicating if system is closed shell."""
-
+        if self.initFlag:
+            return None 
```

Comment:
This contradicts the current docstring. Also, by returning `None`, you've introduced a route for a very subtle bug since usage is probably more akin to `if orbitals.closed_shell`:

```
>>> values = [ None, True, False ]
>>> for value in values:
...     if value:
...             print("Value is true")
...     else:
...             print("Value is false")
... 
Value is false
Value is true
Value is false
```